pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -804,6 +844,15 @@ func loadConfigFile(path string, forceV2 bool) (*parsedConfig, error) {
 		res.shortNameMode = types.ShortNameModeInvalid
 	}
 
+	// Valid wildcarded prefixes must be in the format: *.example.com
+	for i := range res.partialV2.Registries {
```

Comment:
Wouldn’t this be a natural fit for pre-existing `parseLocation`? See the other comment as well.

Reply:
Actually `parseLocation` is used for other things like search; we will need separate validation for prefixes and locations, but either way having it closer to the existing format checks in `postProcessRegistries` would be a bit easier to follow.

Reply:
@vrothberg wdyt?

Reply:
I think we can leave it here until we have figured out what how do validate prefix/location (see other thread).

Reply:
marking this as resolved for now.